apiserver/facade/interface.go

```diff
@@ -87,6 +87,11 @@ type Authorizer interface {
 	// utility func if anything.
 	AuthMachineAgent() bool
 
+	// AuthApplicationAgent returns true if the entity is an application
+	// operator. Doesn't need to be on this interface, should be a
```

Comment:
heed the comment and make it a utility function?

Reply:
If I did it for this, it would need to be done for all others to be consistent which would affect a lot of tests etc. I thought it easiest to keep with what's there and fix as a separate refactoring PR.

Reply:
IMO, this is small enough that we can live with inconsistency for a short time. Please file a bug and leave a TODO at least.